### ISSUE TITLE
Twitter auth no longer working

### ISSUE THREAD

**Issue by icvitkovac** [hapijs/bell#492]
### Runtime

Node

### Runtime version

18.17.1

### Module version

13.0.1

### Any other relevant information

Free twitter developer account that has access to 1.1 API (that includes OAuth)

### What are you trying to achieve or the steps to reproduce?

Basic twitter Oauth 1.0a example:

`const twitterAuthStrategy = {
    name: 'twitter',
    scheme: 'bell',
    // request email scope from twitter

    options: {
        provider: 'twitter',
        password: process.env.BELL_PASSWORD,
        clientId: process.env.TWITTER_API_KEY,
        clientSecret: process.env.TWITTER_API_SECRET,
        isSecure: config.isProduction,
        location: config.baseUrl,
    },
    config:{
        extendedProfile: true,
        getMethod: 'account/verify_credentials.json?include_email=true',
    }
};`

### What was the result you got?

`Parsed Payload: {
0|server   |   errors: [
0|server   |     {
0|server   |       message: 'You currently have access to a subset of Twitter API v2 endpoints and limited v1.1 endpoints (e.g. media post, oauth) only. If you need access to this endpoint, you may need a different access level. You can learn more here: https://developer.twitter.com/en/portal/product',
0|server   |       code: 453
0|server   |     }
0|server   |   ]
0|server   | }
`

### What result did you expect?

It shouldn't cause twitter to throw 403, because as per official twitter documentation, https://api.twitter.com/1.1/users/show should work. 

Even if additional email "scope" is omitted (so, with pure example that you're providing), result is the same. 

So, it should either work for OAuth 1.0a, or it should support OAuth 2.0

**Comment by icvitkovac** [comment]
Yeah, but that doesn't provide an user's email (and Twitter V2 doesn't support that yet in https://api.twitter.com/2/users/me).

So it is essentially broken on both ends, at least that basic (but very important) functionality. 

Re: deprecation - it never says anywhere that it **is** deprecated, including in the link you pasted.

**Comment by pabueco** [comment]
Yes, I only wanted to mention this workaround in case someone does not need the profile/email. But you're right that it's essentially broken.

I didn't say the endpoint was deprecated, just that they deprecated/changed their API access tiers and the endpoint is no longer accessible via the free tier.
